packages/main/src/plugin/extension-loader.ts

```diff
@@ -580,7 +580,11 @@ export class ExtensionLoader {
     return entries
       .filter(entry => entry.isDirectory())
       .filter(directory => directory.name !== 'node_modules')
-      .map(directory => path.join(folderPath, directory.name));
+      .map(directory =>
+        fs.existsSync(path.join(folderPath, directory.name, 'package.json'))
+          ? path.join(folderPath, directory.name)
+          : path.join(folderPath, directory.name, 'packages', 'extension'),
+      );
```

Comment:
We may have extensions having a `package.json` file at the root of the extensions/folder ? like a monorepository ?

in that case it may fail here ?

Reply:
I can give priority, to packages/extension folder in case or monorepo.